ui/src/components/EnrollModal.vue

```diff
@@ -44,6 +46,7 @@
   </v-dialog>
 </template>
 
+
```

Comment:
The syntax checker is complaining about this line, please remove it.

Reply:
Done.